src/WebJobs.Script.WebHost/Security/DefaultKeyValueConverterFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public sealed class DefaultKeyValueConverterFactory : IKeyValueConverterFactory
+    {
+        private static readonly PlaintextKeyValueConverter PlainTextConverter = new PlaintextKeyValueConverter();
+
+        public IKeyValueConverter GetValueConverter(Key key, KeyConversionAction action)
```

Comment:
Not sure I understand the need for KeyConversionAction. I don't think you're event using it in this PR? I'd expect the returned instance to have read/write methods on it, meaning an action enum isn't needed? Perhaps remove the enum until your encryption PR.